grace/io/image_dataset.py

```diff
@@ -74,9 +95,25 @@ def __getitem__(self, idx: int) -> tuple[torch.Tensor, dict]:
             self.image_reader_fn(img_path), dtype=torch.float32
         )
         grace_dataset = read_graph(grace_path)
+        graph = grace_dataset.graph
+
+        # Print original graph label statistics:
+        if self.verbose is True:
```

Comment:
Why using if <value> is True, instead of if <value>? Same question for the lines below.

Reply:
No preference, I only thought that in the installation of the class, the type is already defined, this step might not be necessary but maybe there was another reason.